tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -67,3 +68,76 @@ func TestAuthNPolicy(t *testing.T) {
 		}
 	})
 }
+
+func TestAuthNJwt(t *testing.T) {
+	// V1alpha3 == true implies envoyv2, jwt authn doesn't work for v2 so skip it now.
```

Comment:
Can you add todo to enable this test once fix for v2 is in (link with your PR or issue)

Reply:
done